style.css

```diff
@@ -185,3 +182,389 @@ a:hover {
+  float: left;
+  top: 0;
+}
+
+.title {
+  display: flex;
+  flex-direction: column;
+  width: 375px;
+  height: 107px;
+  align-items: center;
+  justify-content: center;
+  justify-content: space-between;
+  left: 24px;
+}
+
+#recent {
+  width: 327px;
+  height: 10px;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 30px;
+  align-items: center;
+  justify-content: center;
+  color: #1c1a19;
+}
+
+.line {
+  width: 327px;
+  margin-top: 24px;
+}
+
+.frame {
+  display: flex;
+  flex-direction: column;
+  width: 375px;
+  height: 3028px;
+  align-items: center;
+  gap: 24px;
+  position: relative;
+  bottom: 50px;
+}
+
+.featured {
+  justify-content: center;
+  gap: 24px;
+  width: 375px;
+  height: 568px;
+}
+
+.img {
+  width: 327px;
+  height: 237px;
+  background: url('../Images/Work-img1.png') no-repeat;
+  margin-left: 24px;
+}
+
+.block {
+  display: block;
+  flex-direction: column;
+  width: 327px;
+  height: 307px;
+  position: relative;
+  left: 24px;
+  margin-top: 50px;
+}
+
+.multi {
+  display: block;
+  width: 327px;
+  height: 1px;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  align-items: center;
+  color: #091e42;
+  position: relative;
+  bottom: 30px;
+  margin-top: 50px;
+}
+
+.sup {
+  width: 327px;
+  font-family: 'Inter', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  color: #091e42;
+  position: relative;
+  bottom: 40px;
+  margin-top: 50px;
+}
+
+.textblock-tags {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  padding: 0;
+  gap: 12px;
+  width: 297px;
+  height: 40px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  align-self: baseline;
+  margin-top: -158px;
+  list-style-type: none;
+  margin-left: 24px;
+}
+
+.textblock-tags > li {
+  background: #fff;
+  border: 1px solid #8993a4;
+  padding: 10px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+}
+
+.textblock-link {
+  background-color: #ff6b00;
+  color: white;
+  padding: 14px 25px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  bottom: 30px;
+  width: 124px;
+}
+
+#project {
+  order: 5;
+  border-radius: 8px;
+}
+
+.proj {
+  display: flex;
+  flex-direction: row;
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+  list-style-type: none;
+  align-items: center;
+  justify-content: space-between;
+  position: relative;
+  margin-top: 0;
+  width: 200px;
+  height: 40px;
+  margin-left: 24px;
+}
+
+.frame43 {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  gap: 24px;
+  width: 375px;
+  height: 2436px;
+  position: relative;
+  left: 24px;
+  margin-top: 12px;
+}
+
+.content-defaults {
+  display: flex;
+  flex-direction: column;
+  width: 327px;
+  height: 386px;
+  background: url('../Images/Img-Placeholder.png') no-repeat;
```

Comment:
[OPTIONAL] **There is an extra dot `.` in your image URL kindly make sure you keep one dot. 
Your code should look like this :arrow_down:**

` list-style-image: url("./Images/Rectangle55.png");`  **Notice that I removed one dot from the URL path**